src/sentry/api/endpoints/sentry_app_errors.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+# from rest_framework.response import Response
+
+from sentry.api.bases import SentryAppBaseEndpoint, SentryAppStatsPermission
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.api.utils import get_date_range_from_params
+from sentry.models import SentryAppWebhookError
+
+
+class SentryAppErrorsEndpoint(SentryAppBaseEndpoint):
+    permission_classes = (SentryAppStatsPermission,)
+
+    def get(self, request, sentry_app):
+        """
+        :qparam float start
+        :qparam float end
+        """
+        start, end = get_date_range_from_params(request.GET, optional=True)
```

Comment:
@nolachen What happens if someone has invalid values for the date range? I see in other places we catch the `InvalidParams` error but not here.